inc/notificationeventmailing.class.php

```diff
@@ -285,8 +285,11 @@ static public function send(array $data) {
             }
 
             $mmail->ClearAddresses();
-            $current->update(['id'        => $current->fields['id'],
-                                'sent_try' => $current->fields['sent_try']+1]);
+            $current->update([
+                'id'        => $current->fields['id'],
+                'send_time' => date("Y-m-d H:i:s", strtotime('+2 minutes')), //Delay 2 minutes to try again
```

Comment:
Just a question... If your crontab is ran each 5 minutes; the mecanism will not be efficient; am I wrong?

I also guess this should made configurable (from an overridable constant).

Reply:
It will be efficient anyway, it will try to send the emails again, but in the next run, it will send the remaining emails instead of trying the wrong emails again

Reply:
Can I create the option to allow setting this

Reply:
Each entry in the queue should be tried only once per run; if you have something different this is probably a  bug.
As of the option, I had in mind a constant in `inc/define.php`; not from the configuration, but this can be ok as well.

Reply:
But each list item is tested only once, will only be tested again the next run.
I will try to make an example

Reply:
Run step: 3

| 10:00 | 10:05 | 10:10 | 10:15 | 10:20 |
|---|---|---|---|---|
| 1W 10:00  | 4W 10:00 | 5W 10:04 | 1W 10:07 | 5W 10:12 |
| 2W 10:00  | 1W 10:02 | 6R 10:04 | 2W 10:07 | 1W 10:17 |
| 3R 10:00  | 2W 10:02 | 4W 10:07 | 4W 10:12 | 2W 10:17 |
| 4R 10:00  | 5W 10:04 | 1W 10:07 | 5W 10:12 | 4W 10:17 |
| 5W 10:04  | 6R 10:04 | 2W 10:07 |  |   |
| 6R 10:04 |   |   |   |   |

W - Wrong
R - Right